simple_network_sim/network_of_populations/report.py

```diff
@@ -0,0 +1,44 @@
+from simple_network_sim.network_of_populations import logger
+
+
+# NotCurrentlyInUse
```

Comment:
Why are we keeping this stuff if it is not in use? Might it return to use? Is it worth just deleting it and leaving it in the history?

Reply:
Those were functions that @magicicada created in the very beginning. I was under the impression that she used them during development of the model, that's why I've never removed them. But If that's not the case, I think we should delete.